CHANGELOG.md

```diff
@@ -34,6 +34,10 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 - Removed OMIM genes panel
 - Make genes panel, pinned variants panel, causative variants panel and ClinVar panel scrollable on case page
 
+## [4.37]
+### Added
+- On login, the browser userAgent and screen size are saved anonymously to understand which browsers are used by Scout users
```

Comment:
```suggestion
- On login, OS, browser userAgent and screen size are saved anonymously to understand which browsers are used by Scout users
```

Reply:
Is this more correct?

Reply:
in the browser user agent both OS and browser information are included. Maybe I can write
`On login, OS, browser version and screen size are saved anonymously to understand how users are using Scout`